spec/lib/rbac/filterer_spec.rb

```diff
@@ -74,6 +74,12 @@
         expect(results).to match_array [owned_vm]
       end
 
+      it "with :extra_cols finds Service" do
+        FactoryGirl.create :service, :evm_owner => owner_user
+        results = described_class.search(:class => "Service", :extra_cols => [:owned_by_current_user]).first
+        expect(results.first.attributes["owned_by_current_user"]).to be false
```

Comment:
@jrafanie So, this is just a `virtual_attribute` brought in by the `OwnershipMixin`:

https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/ownership_mixin.rb#L23-L26

And one that I know, for certain, exists with `arel` support in the DARGA branch.  Mostly confirming here that we can call `Rbac.search` with `extra_cols`, and it will append the `arel` to the select clause.  Scopes in the past were the only way to do this, and it would be `nil` otherwise if that attribute didn't exist.